Problems with DIRAC nightly

The DIRAC job object seems to be struggling in the latest nightlies:

```
    @pytest.yield_fixture(scope='class')
    def dirac_job(load_config):
```
... Leads to:
```
>       get_file_lfn = output_data_info['getFile.dst']['LFN']
E       KeyError: 'getFile.dst'
```

As a result a lot of the tests fall over. I'll have a look at how we've arrived here and see if it's obvious.

--- a/python/GangaDirac/test/Unit/DiracAPI/TestDiracCommands.py
+++ b/python/GangaDirac/test/Unit/DiracAPI/TestDiracCommands.py
@@ -130,7 +130,7 @@ def dirac_job(load_config):
 
     output_data_info = execute('getOutputDataInfo("%s")' % job_id)
     logger.info(output_data_info)
-    while not output_data_info.get('OK', True):
+    while not output_data_info:
         time.sleep(5)
         output_data_info = execute('getOutputDataInfo("%s")' % job_id)
         logger.info("\n%s\n", output_data_info)

